interfaces/python/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ else()
 
             set(CMAKE_SWIG_FLAGS "-py3")
 
+            if("${SWIG_VERSION}" VERSION_GREATER "4.0.0")
```

Comment:
we should add this to the Java CMakeLists as well

Reply:
Resolved in 012e2867bdb64e04c957eb4cbd0e82b22ce6e441.